internal/cmd/util/validation_test.go

```diff
@@ -0,0 +1,70 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/hetznercloud/cli/internal/testutil"
+)
+
+func TestValidateExact(t *testing.T) {
```

Comment:
As the structure of all these tests is exactly the same, you could use [Table Driven Tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) to simplify adding new test cases.